duckduckgo/duckduckgo-privacy-extension
# Fix service worker requests

**Reviewer:** @kzar  or @sammacbeth



## Description:



## Steps to test this PR:

1. Go to https://www.vice.com/en
2. Check that npttech.com isn't blocked

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
